project_event/views/project_task_view.xml

```diff
@@ -44,6 +44,8 @@
                             <field name="project_id" string="Event" options="{'no_create_edit': True}"/>
                             <field name="responsible_id" required="1"/>
                             <field name="partner_id"/>
+                            <field name="client_type"/>
+                            <field name="sector_id"/>
```

Comment:
@jananjoy the sector field should be visible only if the client_type is not empty

Reply:
@rimbendhaou Done